src/components/app/AppStatus.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { AppStatusType } from './types'
+
+export default function AppStatus({ appStatus }: AppStatusType) {
+    const appStatusLowerCase = appStatus?.toLowerCase()
+    const isNotDeployed = appStatusLowerCase === 'notdeployed'
```

Comment:
please create constants for `notdeployed`

Reply:
done